tool.go

```diff
@@ -143,7 +143,20 @@ func main() {
 					}
 					if len(pkgs) == 1 { // Only consider writing output if single package specified.
 						if pkgObj == "" {
-							pkgObj = filepath.Base(pkg.ImportPath) + ".js"
+							importPath := filepath.Base(pkg.ImportPath)
+							if importPath == "." {
+								pkgObj = filepath.Base(currentDirectory) + ".js"
+							} else if importPath == ".." {
+								// allow for build requests like "build .. or build ../.."
+								path := currentDirectory
+								count := strings.Count(pkg.ImportPath, "..")
```

Comment:
I hate to keep commenting on this... It's such a corner case. But I don't think this is right. It would incorrectly count for a path like `foo..bar/..`.

Reply:
You are tenacious.

Reply:
I was a bit confused by what you might mean, thinking that perhaps there are other ways of building that I am unfamiliar with (not to say there are not), but if you mean to say "gopherjs build over..here/.." then I just tried that and it works for me with the patch as-is.

Reply:
I think that means your new codepath is never being executed.  Because "over..here/.." clearly contains two instances of "..".

When I execute similar tests, I also get the right output, but I also see that `pkg.ImportPath` has already had the trailing "/.." removed.  In fact, I can't even get `gopherjs build ..` to break.  I expect this is related to the operating system and/or shell I'm using (Linux/bash).  Since @shurcooL  was able to produce the problem with "..", maybe he can provide further details (although I'm pretty sure he's using OSX) and/or test some other ".." combinations.